core/api/src/app/authentication/login.ts

```diff
@@ -57,6 +58,7 @@ import { ErrorLevel } from "@/domain/shared"
 import { consumeLimiter } from "@/services/rate-limit"
 import { RateLimitConfig } from "@/domain/rate-limit"
 import { RateLimiterExceededError } from "@/domain/rate-limit/errors"
+import { kratosAdmin } from "@/services/kratos/private"
```

Comment:
The file you are importing is flagged as `private`. That means you are not allowed to depend on it! That's what private means.
Only code inside `@/services/kratos/` should be allowed access.. unfortunately TS doesn't support module private files / functions so we have to enforce it manually.